include/npc.h

```diff
@@ -190,14 +190,14 @@ typedef struct EnemyDrops {
 enum TerritoryShape { SHAPE_CYLINDER, SHAPE_RECT };
 
 typedef struct {
-    /* 0x00 */ s32 unk_00;
+    /* 0x00 */ s32 skipPlayerDetectChance;
     /* 0x04 */ enum TerritoryShape shape;
     /* 0x08 */ s32 pointX;
     /* 0x0C */ s32 pointZ;
     /* 0x10 */ s32 sizeX;
     /* 0x14 */ s32 sizeZ;
-    /* 0x18 */ f32 unk_18;
-    /* 0x1C */ s16 unk_1C;
+    /* 0x18 */ f32 halfHeight;
+    /* 0x1C */ s16 unk_1C;  // 1 = ignore partner hiding (bow/sushie dont work) | 2 = ignore elevation
```

Comment:
Should rename this field & make a flags enum for it

Reply:
Again, not enough uses of this field to give it a definitive name yet. Need to decomp more enemy AI types.